app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt

```diff
@@ -79,6 +82,14 @@ class LoginActivity : BaseActivity() {
         return validCredentials
     }
 
+    /**
+     * This method closes the virtual keyboard on the device.
+     * */
+    private fun hideKeyboard(){
```

Comment:
@Hardik0403 nice work, but can you please implement this function inside Utils class or create new utility file as we may need functionality at other places in the application